servant-client/src/Servant/Client.hs

```diff
@@ -417,6 +417,12 @@ instance HasClient api => HasClient (IsSecure :> api) where
   clientWithRoute Proxy req baseurl manager =
     clientWithRoute (Proxy :: Proxy api) req baseurl manager
 
+instance HasClient subapi =>
+  HasClient (WithNamedConfig name config subapi) where
```

Comment:
Shouldn't there be? I guess it doesn't have to be in this PR, since it's backwards-compatible and we don't want to hold this up.